### DISCUSSION TITLE
Official quarto markdown dialect style guide?

### DISCUSSION THREAD

**Discussion by MichelNivard** [quarto-dev/quarto-cli#11233] [Q&A]
### Description

Beyond the (amazingly comprehensive!) quarto website, is there an official quarto markdown dialect manual? I am building a WYSIWYG R evaluator for obsidian markdown and want to make it as close to fully compatible with the quarto markdown dialect as I possibly can (maximize interoperability).

**Comment by cscheid** [comment]
No, there unfortunately isn't. This is on our radar for 2025, but we don't have official anything to share.

Two important concepts you should know about are:

- code blocks with the `` ```{lang}... `` syntax. This isn't available in Pandoc but we support it
  - code options declared as a YAML object in the "code block front matter". This:
    
    ````
    ```{python}
    #| echo: true
    ```
    ````
    
    This syntax is carefully designed to be a valid comment in the host language, so if you use C++, you'd have
    
    ````
    ```{c++}
    //| echo: true
    ```
    ````
- shortcodes. Also not available in Pandoc by default but supported in Quarto. Unfortunately, we don't have a clean grammar/parser to share either.

**Reply by mcanouil** [reply]
"Everything else" is in https://pandoc.org/MANUAL.html, in particular https://pandoc.org/MANUAL.html#pandocs-markdown
